### PR TITLE
Added features to manage timeoff records of the employee

### PR DESCRIPTION
**Pull request by Hamsitha-Challagundla**
Added features to manage timeoff records of the employee

### CODE REVIEW

File: NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/model/Payroll.java [hunk 1]
```diff
@@ -1,11 +1,8 @@
 package com.nullterminators.project.model;
 
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.Id;
-import jakarta.persistence.Table;
-import jakarta.persistence.GenerationType;
+import jakarta.persistence.*;
```
Comment (madZ604) [inline NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/model/Payroll.java:3]: Why did you replace with .*? Checkstyle does not support .* imports

File: NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/controller/EmployeeProfileController.java [hunk 2]
```diff
@@ -2,26 +2,21 @@
...
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.DeleteMapping;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```
Comment (madZ604) [inline NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/controller/EmployeeProfileController.java:8]: Why did you replace with .\*? Checkstyle does not support .\* imports

File: NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/model/EmployeeProfile.java [hunk 3]
```diff
@@ -1,18 +1,14 @@
 package com.nullterminators.project.model;
 
-import jakarta.persistence.Column;
-import jakarta.persistence.Entity;
-import jakarta.persistence.GeneratedValue;
-import jakarta.persistence.GenerationType;
-import jakarta.persistence.Id;
-import jakarta.persistence.Table;
+import jakarta.persistence.*;
```
Comment (madZ604) [inline NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/model/EmployeeProfile.java:3]: Same .*

File: NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/model/Payroll.java (hunk 1 above)
Comment (Hamsitha-Challagundla) [reply NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/model/Payroll.java]: Corrected it!

File: NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/controller/EmployeeProfileController.java (hunk 2 above)
Comment (Hamsitha-Challagundla) [reply NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/controller/EmployeeProfileController.java]: Corrected it!

File: NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/model/EmployeeProfile.java (hunk 3 above)
Comment (Hamsitha-Challagundla) [reply NullTerminatorsTeamProject/src/main/java/com/nullterminators/project/model/EmployeeProfile.java]: Corrected it!

File: NullTerminatorsTeamProject/pom.xml [hunk 4]
```diff
@@ -1,165 +1,184 @@
...
-			<version>3.0.1</version>
-		</dependency>
-		<dependency>
-			<groupId>org.hibernate.validator</groupId>
-			<artifactId>hibernate-validator</artifactId>
-			<version>7.0.0.Final</version>
-		</dependency>
-		
-	</dependencies>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (Samhit-Chowdary) [inline NullTerminatorsTeamProject/pom.xml:3]: these changes are not required, right?

**Pull request closed without merging**